Gmplot-Track the Route/README.md

```diff
@@ -0,0 +1,41 @@
+# Description 
+- You have been provided with a [`CSV`](https://www.bigcommerce.com/ecommerce-answers/what-csv-file-and-what-does-it-mean-my-ecommerce-business/#:~:text=A%20CSV%20is%20a%20comma,Microsoft%20Excel%20or%20Google%20Spreadsheets.)(comma separated value) file in which the **latitudes** and **longitudes** of a specific area are written
+_So_ we can track the whole route of a person by reading values of that file on Google Maps and create an Image containing route or location .
+_OR_ Just point the location on the map .
+
+# Details 
+- API Used [`gmplot`](https://github.com/gmplot/gmplot/wiki)
+- Library Used [`csv`](https://docs.python.org/3/library/csv.html)
+
+# What to do
+1. Take CSV file input **or** any latitudes or longitudes
+2. plot the map 
+3. give output in html file format
+
+# Commands to use 
+
+| Command | README |
+| ------ | ------ |
+| `pip install gmplot` | [Gmplot/PyPI](https://pypi.org/project/gmplot/) |
+
+# CSV File Data

```

Comment:
Now you can remove this section and corresponding lat-long values

Reply:
Done ✅